stage1_fly/enter/main.go

```diff
@@ -47,6 +55,33 @@ func getRootDir(pid string) (string, error) {
 	return os.Readlink(rootLink)
 }
 
+func getPodManifest() (*schema.PodManifest, error) {
```

Comment:
I think this has quite a lot in common with `common/types.LoadPod()` and can probably just use that, or refactor/expose/move some logic from there.